data/store/scripts/outfits.lua

```diff
@@ -0,0 +1,79 @@
+-- Load outfits
+local outfits = {}
+
+for line in io.lines('data/XML/outfits.xml') do
+    local sex = tonumber(line:match('type.-=[\'"](%d-)[\'"]'))
+    local lookType = tonumber(line:match('looktype.-=[\'"](.-)[\'"]'))
+    local name = line:match('name.-=[\'"](.-)[\'"]')
+    local enabled = line:match('enabled.-=[\'"](.-)[\'"]') == 'yes'
+    local unlocked = line:match('unlocked.-=[\'"](.-)[\'"]') == 'yes'
```

Comment:
Regexing a XML file: please DON'T YOU EVER DARE.

[Here's why](http://stackoverflow.com/a/1732454/1940360).